package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "systelab-components",
-  "version": "5.3.7",
+  "version": "5.4.7",
```

Comment:
After the 5.3.7, the following version should be 5.4.0.

Reply:
ok, it's done.